lms/static/sass/views/_homepage.scss

```diff
@@ -0,0 +1,168 @@
+
+    .courses-listing {
+      margin: 0;
+      padding: 0;
+      list-style: none;
+
+      .courses-listing-item {
+        @include fill-parent();
+        max-height: $course-card-height;
+        margin: ($baseline*0.75) 0;
+
+        @include media($medium) {
+          @include span-columns(4); // 4 of 8
+          @include omega(2n);
+        }
+
+        @include media($large) {
+          @include span-columns(4); // 4 of 12
+          @include omega(3n);
+        }
+
+        @include media($huge) {
+          @include span-columns(3); // 3 of 12
+          @include omega(4n);
+        }
+      }
+    }
+
+    .course {
+      @include box-sizing(border-box);
+      @include transition(all $tmg-f3 linear 0s);
+      box-shadow: 0 1px 10px 0 $black-t0, inset 0 0 0 1px $white-t3;
+      margin-bottom: ($baseline*1.5);
+      position: relative;
+      border-bottom: 3px solid $action-primary-bg;
+      background: $body-bg;
+      width: 100%;
+
+      .course-image {
+        .cover-image {
+          height: $course-image-height;
+          overflow: hidden;
+
+          // places the shadow on top of the course image while hovering over the course card
+          &:before {
+            @include left(0);
+            @extend %ui-depth1;
+            position: absolute;
+            top: 0;
+            opacity: 0;
+            background: $black;
+            width: 100%;
+            height: $course-image-height;
+            content: '';
+          }
+
+          img {
+            width: 100%;
+            height: auto;
+          }
+
+          .learn-more {
+            @include box-sizing(border-box);
+            @include left($learn-more-horizontal-position);
+            @include line-height(28);
+            @extend %ui-depth2;
+            position: absolute;
+            top: ($baseline*2.75);
+            opacity: 0;
+            border: 3px solid $white;
+            border-radius: 3px;
+            padding: 0 $baseline;
+            width: ($baseline*10);
+            height: ($baseline*2.5);
+            text-align: center;
+            color: $white;
+          }
+        }
+      }
+
+      .course-info {
+        height: $course-info-height;
+
+        .course-organization, .course-code, .course-date {
+          @extend %t-icon6;
+          color: $black;
+        }
+
+        .course-organization, .course-code, .course-title {
+          display: block;
+        }
+
+        .course-organization {
+          @include line-height(11);
+          padding: ($baseline*0.5) ($baseline*0.75) ($baseline*0.1) ($baseline*0.75);
+        }
+
+        .course-code {
+          @include line-height(16);
+          padding: 0 ($baseline*0.75);
+        }
+
+        .course-title {
+          @include line-height(16);
+          @extend %t-icon4;
+          margin: ($baseline*0.25) 0 ($baseline*1.75) 0;
+          padding: 0 ($baseline*0.75);
+          height: $course-title-height;
+          color: #701f3f;
```

Comment:
Please change this value to use the ``$link-color`` platform Sass variable (https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order)